.drone.star

```diff
@@ -0,0 +1,100 @@
+OC_CI_PHP = "owncloudci/php:%s"
+OC_UBUNTU = "owncloud/ubuntu:20.04"
+PLUGINS_GITHUB_RELEASE = "plugins/github-release"
+
+DEFAULT_PHP_VERSION = "8.1"
+
+dir = {
+    "base": "/var/www/owncloud",
+}
+
+config = {
+    "branches": [
+        "main",
+    ],
+    "codestyle": True,
+    "phpstan": True,
+    "phan": {
+        "multipleVersions": {
+            "phpVersions": [
+                DEFAULT_PHP_VERSION,
+                "8.2",
+            ],
+        },
+    },
+    "phpunit": True,
+}
+
+
+def main(ctx):
+    return tests(
+        ctx,
+        [
+            ["codestyle", "make test-php-style"],
+            ["phpstan", "make test-php-phpstan"],
+            ["phan", "make test-php-phan"],
+        ],
+    ) + phpunit(ctx, [8.1, 8.2])
+
+
+def tests(ctx, tests):
+    pipelines = []
+    for index, test in enumerate(tests):
+        if test[0] not in config:
+            return pipelines
+        pipelines += [
+            {
+                "kind": "pipeline",
+                "name": test[0],
+                "steps": [
+                    {
+                        "name": str(index)+test[0],
+                        "image": OC_CI_PHP % DEFAULT_PHP_VERSION,
+                        "commands": [
+                            "composer install",
+                            test[1],
+                        ],
+                        "trigger": {
+                            "ref": [
+                                "refs/head/main",
+                                "refs/pull/**",
+                                "refs/tags/**",
+                            ],
+                        },
+                    },
+                ],
+            }
+        ]
+    return pipelines
+
+
+def phpunit(ctx, phpVersion):
```

Comment:
```suggestion
def phpunit(ctx, phpVersions):
```

It will be more obvious that multiple versions can be passed if we call this `phpVersions`